chemprop/nn/loss.py

```diff
@@ -23,13 +22,12 @@
     "DirichletMixin",
     "BinaryDirichletLoss",
     "MulticlassDirichletLoss",
-    "_ThresholdMixin",
     "SIDLoss",
     "WassersteinLoss",
 ]
 
 
-class LossFunction(ABC, ReprMixin):
+class LossFunction(ABC, nn.Module):
     def __call__(
```

Comment:
we'll probably have to rename this to `forward()` and the original `forward()` method to `_forward()` or `_calc_unreduced_loss()` to adhere to the `nn.Module` interface.